src/any_agent/frameworks/any_agent.py

```diff
@@ -41,21 +40,46 @@ def __init__(
     ):
         self.config = config
         self.managed_agents = managed_agents
+
         self._mcp_servers: list[MCPServerBase] = []
-        self._last_tracer: Tracer | None = (
-            None  # holds the trace from the most recent run
-        )
-        self._tracing_config = tracing
```

Comment:
Whoops I missed this during the initial review. Making this change means that now all the `agent.run()` commands from an agent are going to get merged into a single trace. That's going to be an issue for people that are running the agent in a server or want to have multiple runs happening in a single instantation of the agent.

```   
 result = agent.run("What's the best agent framework")
 result2 = agent.run("What's the best agent framework")
```
result2 is going to be result and result2 appended on top of it